hw/ip_templates/alert_handler/rtl/alert_handler_ping_timer.sv.tpl

```diff
@@ -49,7 +49,7 @@ module ${module_instance_name}_ping_timer import ${module_instance_name}_pkg::*;
   output logic                     esc_ping_fail_o     // any of the esc senders failed
 );
 
-  localparam int unsigned IdDw = $clog2(NAlerts);
+  localparam int unsigned IdDw = prim_util_pkg::vbits(NAlerts + 1);
```

Comment:
I'm a bit confused: Aren't there only NAlerts items, with the index 0 being the first one? Where does the need to represent +1 items come from?